src/mumble/Log.cpp

```diff
@@ -374,7 +370,8 @@ QString Log::imageToImg(QImage img) {
 		imgwrite.write(img);
 	}
 
-	while ((qba.length() >= 65536) && (quality > 0)) {
+	const int safety = 1024;
+	while ((static_cast<unsigned int>(qba.length() * 2 + safety) >= g.uiImageLength) && (quality > 0)) {
```

Comment:
`static_cast<unsigned int>(qba.length() * 2 + safety)` overflows when qba.length() is near INT_MAX.
Can we do something about that?